packages/open-api-render-engine/README.md

```diff
@@ -7,20 +7,31 @@ The OpenAPI render engine is based on the [Swagger UI](https://github.com/swagge
 ## Installation
 
 - using [`npm`](https://www.npmjs.com/)
-  ``` bash
+
+  ```bash
   npm i @kyma-project/dc-open-api-render-engine
   ```
 
 - using [`yarn`](https://yarnpkg.com/en/)
-  ``` bash
+
+  ```bash
   yarn add @kyma-project/dc-open-api-render-engine
   ```
-  
+
 ## Configuration
 
-| Name | Required | Type | Default value | Description |
-|---|---|---|---|---|
-| `plugins` | No | `any` | `[]` | Defines custom plugins for the [Swagger UI](https://github.com/swagger-api/swagger-ui) component. Read [this](https://github.com/swagger-api/swagger-ui/blob/master/docs/customization/plugin-api.md) document for more details. |
-| `schemaUrl` | No | `any` | `{}` | Defines the custom schema for the [SwaggerUI](https://github.com/swagger-api/swagger-ui) component. |
+| Name        | Required | Type  | Default value | Description                                                                                                                                                                                                                      |
+| ----------- | -------- | ----- | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `plugins`   | No       | `any` | `[]`          | Defines custom plugins for the [Swagger UI](https://github.com/swagger-api/swagger-ui) component. Read [this](https://github.com/swagger-api/swagger-ui/blob/master/docs/customization/plugin-api.md) document for more details. |
+| `schemaUrl` | No       | `any` | `{}`          | Defines the custom schema for the [SwaggerUI](https://github.com/swagger-api/swagger-ui) component.                                                                                                                              |
+
+> **NOTE**: To learn how to pass options of a render engine, read [this](../../docs/props/render-engines.md#pass-global-options) document.
+
+### Styles
```

Comment:
Please only mention about our styling inside package and add additional info in blockquote block about fonts from fiori.

Reply:
Please also write about default styling from `swagger-ui` and add appropriate import for that.

Reply:
Swagger-ui is unusable without those defaults, so I think we shouldn't even consider leaving it as a choice for a user